Project/src/main/java/com/github/lgooddatepicker/components/DatePickerSettings.java

```diff
@@ -78,6 +79,7 @@
         CalendarTextWeekdays(Color.black),
         CalendarTextWeekNumbers(Color.black),
         TextClearLabel(new JLabel().getForeground()),
+        TextHoverLabel(Color.black),
```

Comment:
This color never used in the rest of the Code, I think something is missing in your code here.

Reply:
Oh sorry this was my mistake. I got confused by the `setForeground` terminology, the foreground is the color of the text. and is actually used. Please disregard this comment.